packages/fetch-http-handler/src/fetch-http-handler.ts

```diff
@@ -83,7 +83,7 @@ export class FetchHttpHandler implements HttpHandler {
           transformedHeaders[pair[0]] = pair[1];
         }
 
-        const hasReadableStream = response.body !== undefined;
+        const hasReadableStream = response.body != undefined;
```

Comment:
nit: Add a comment that this checks for undefined as well as null.
```suggestion
        // Check for undefined as well as null.
        const hasReadableStream = response.body != undefined;
```